docs/authentication/social-connections/microsoft.mdx

```diff
@@ -21,120 +21,119 @@ description: Learn how to allow users to sign into your Clerk app with their Mic
+  Once your Microsoft Entra ID app is created, or once you select your app from the **Microsoft Azure portal**, you'll be taken to its **Overview** page.
 
   1. From your app's overview, copy the **Application (client) ID** and save it somewhere secure. It's required for connecting your Microsoft Entra ID app to your Clerk app.
-  1. On this same page, under **Client credentials**, select **Add a certificate or secret** to generate a Client Secret. You'll be taken to the **Certificate & secrets** page.
-  1. Select the **New client secret** button. In the modal that opens, add a description and set an expiration time for your secret.
+  1. On this same page, under **Client credentials**, select **Add a certificate or secret** to generate a Client Secret. You'll be redirected to the **Certificate & secrets** page.
+  1. Select the **New client secret** button. In the panel that opens, add a description and set an expiration time for your secret. Select **Add**.
      > [!IMPORTANT]
      > When your secret expires, your social connection will stop working until you generate a new client secret and add it to your Clerk app.
   1. Copy your new client secret's **Value** and save it somewhere secure. You'll add it to your Clerk application later, alongside your client ID.
   1. Leave this tab open.
 
   ### Connect your Entra ID app and get your redirect URI
 
-  1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections).
+  1. Navigate to the [**Clerk Dashboard**](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections).
   1. In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**.
   1. Select the **Add connection** button, and select **For all users**.
-  1. In the **Choose provider** dropdown, select **Microsoft**.
+  1. In the **Choose provider** dropdown, search for and select **Microsoft**.
   1. Ensure that both **Enable for sign-up and sign-in** and **Use custom credentials** are toggled on. Then:
-     - Under **Client ID**, add the value you copied from **Application (client) ID** in the Microsoft Entra ID dashboard.
-     - Under **Client Secret**, add the client secret value you generated in the Microsoft Entra ID dashboard.
+     - Under **Client ID**, paste the value you copied from **Application (client) ID** the Microsoft Azure portal.
+     - Under **Client Secret**, paste the client secret value you generated from Microsoft Entra ID.
      - Copy the **Authorized redirect URI**. You need it for the final step to configure your Entra ID app.
      - Select **Add connection**.
 
   ### Enable OpenID
 
-  To connect your Clerk app to your Microsoft app, you must set the **Authorized redirect URI** in your Microsoft Entra ID dashboard.
+  To connect your Clerk app to your Microsoft app, you must set the **Authorized redirect URI** in your Microsoft Entra Azure portal.
```

Comment:
I think we were using Microsoft Azure portal and not Microsoft Entra Azure portal

```suggestion
  To connect your Clerk app to your Microsoft app, you must set the **Authorized redirect URI** in your Microsoft Azure portal.
```